client/asset/eth/multirpc.go

```diff
@@ -697,7 +696,7 @@ func createAndCheckProviders(ctx context.Context, walletDir string, endpoints []
 		if len(providers) != len(unknownEndpoints) {
 			return providersErr(providers)
 		}
-		if err := checkProvidersCompliance(ctx, providers, net, log); err != nil {
+		if err := checkProvidersCompliance(ctx, providers, net, dex.Disabled /* logger is for testing only */); err != nil {
```

Comment:
Could do this when calling `createAndCheckProviders`, but that's only used in product code, whereas the tests use `checkProvidersCompliance` directly.
Happy to change.